pkgs/development/python-modules/paramiko/default.nix

```diff
@@ -14,20 +13,13 @@
 
 buildPythonPackage rec {
   pname = "paramiko";
-  version = "2.7.1";
+  version = "2.7.2";
 
   src = fetchPypi {
     inherit pname version;
-    sha256 = "920492895db8013f6cc0179293147f830b8c7b21fdfc839b6bad760c27459d9f";
+    sha256 = "7f36f4ba2c0d81d219f4595e35f70d56cc94f9ac40a6acdf51d6ca210ce65035";
   };
 
-  patches = [
-    # fix RSA key loading with cryptography 3.1, remove >2.7.1
-    (fetchpatch {
-      url = "https://github.com/paramiko/paramiko/commit/81064206bf3cec2ca4372257ff138481e1227b91.patch";
-      sha256 = "01b87ffgyvd6rilp1w1kf7lk29z706ch39nwl21ifklqpjhmazww";
-    })
-  ];
   checkInputs = [ invoke pytest mock pytest-relaxed ];
```

Comment:
We could move to pytestCheckHook here at some point.

Reply:
sure, if there's issues when updating again, we can revist this, but I don't think it's harming anything as tests are being ran.